MovieApp/Screens/Home/Views/PreviewCell/PreviewCell.swift

```diff
@@ -7,10 +7,65 @@
 
 import UIKit
 
+protocol clickedCell: AnyObject {
+	func goDetailView(celldata: MovieViewModel)
+}
+
 class PreviewCell: UITableViewCell {
+	// MARK: - Iboutlets
+	@IBOutlet weak var previewCollectionView: UICollectionView!
+	// MARK: - Variables
+	weak var delegate: clickedCell?
+	var previewData = [MovieViewModel]() {
+		didSet {
+			previewCollectionView.reloadData()
+		}
+	}
 	// MARK: - Life cycles
 	override func awakeFromNib() {
 		super.awakeFromNib()
-		// Initialization code
+		previewCollectionView.delegate = self
+		previewCollectionView.dataSource = self
+		previewCollectionView.register(UINib(nibName: ConstantsCell.previewCollectionCell, bundle: Bundle.main), forCellWithReuseIdentifier: ConstantsCell.previewCollectionCell)
+		self.backgroundColor = .black
+	}
+	// MARK: - Life cycles
+	func configure(preview: [MovieViewModel]) {
```

Comment:
```suggestion
	func configure(preview: [IMovieViewModel]) {
```

@minh5896 : Don't forget SOLID, replace by interface

Reply:
fix complete